src/cert-provider/cert-provider.sh

```diff
@@ -179,12 +179,12 @@ while ! waitForOnline "${ACME_DOMAINS[0]}"; do
 done
 
 if ! lastAcquiredCertFor "production"; then
-    acquireCertificate "staging" || logErrorAndStop "Unable to acquire a staging certificate."
-    waitToSeeStagingCert || logErrorAndStop "Unable to detect certificate change over. Cannot issue a production certificate."
+    #acquireCertificate "staging" || logErrorAndStop "Unable to acquire a staging certificate."
+    #waitToSeeStagingCert || logErrorAndStop "Unable to detect certificate change over. Cannot issue a production certificate."
```

Comment:
Thanks for putting attention to this and trying to understand this. As far as I understood, PR #131 updated the acme.sh script and also the staging certificates which seem to be signed by the expired root certificate.

As far as I understand the discussion linked by @ipm79 (https://community.letsencrypt.org/t/staging-doctored-durian-root-ca-x3-is-expired-breaks-test-environment/145689/4), the it's comment #4 says that the staging environment is an environment which gets upcoming changes and exists explicitly to make ACME client implementers aware that an upcoming change is going to break their client, tests or use case.

Later in the discussion, there is also word that ACME clients which break with the staging environment and are not fixed should not be used as they are going to be broken with production later as well. That reminds me that the acme.sh updated by #131 is meanwhile also no longer the lastes version of the acme.sh, so we may update that again to the latest version and look of the latest versions of the staging certs updated in #131.

If that fails, the discussion proposes to use the alternate chain instead, or we could use the workaround from #108 (skip the staging steps). The latter worked without staging for @ipm97 and other people in the balena forum.

I'm rebasing the commit now and move prolonging the timeouts a bit into the 1st commit, then the workaround from #108.

Reply:
I see, thank you for the explanation. So the staging environment in question is Let's Encrypt's, not any openBalena staging environment. That's reassuring. :-) It looks to me like this PR's `Change-type` footer should be 'patch' rather than 'minor'.

Reply:
Until I can investigate further, here is a quick hint on how to run the certificate updater with debug manually to check any issue:
```bash
docker exec -it openbalena_cert-provider_1 bash
/root/.acme.sh/acme.sh --cron --home "/root/.acme.sh" --debug
```